Framework/DataHandling/test/LoadEventNexusTest.h

```diff
@@ -1057,6 +1057,27 @@ class LoadEventNexusTest : public CxxTest::TestSuite {
     TS_ASSERT_EQUALS(eventWS->detectorInfo().size(), 921)
   }
 
+  void test_monotonically_increasing_tofs() {
+    const std::string file = "CG2_monotonically_increasing_pulse_times.nxs.h5";
+    LoadEventNexus alg;
+    alg.setChild(true);
+    alg.setRethrows(true);
+    alg.initialize();
+    alg.setProperty("Filename", file);
+    alg.setProperty("OutputWorkspace", "dummy_for_child");
+    alg.setProperty("NumberOfBins", 1);
+    alg.execute();
+    Workspace_sptr ws = alg.getProperty("OutputWorkspace");
+    auto eventWS = std::dynamic_pointer_cast<EventWorkspace>(ws);
+    TS_ASSERT(eventWS);
+    const int expectedNumberEvents = 32494;
```

Comment:
Makes little to no difference in this case, but changing this line to
```suggestion
    constexpr int expectedNumberEvents{32494};
```
is a more modern way to declare this variable. It is a good habit to develop.